client/src/components/Dashboard/LiveTicker.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import axios from 'axios'
+import TickerStar from './TickerStar'
+import { Loading, Row, TickerContainer, StockSymbol, Star } from '../Styles/Dashboard/LiveTickerStyles' 
```

Comment:
We can definitely change the styling of this after we get all of the other functionality working in the coming weeks. I think it should be a lot more compact than it is now (similar to how it is on the landing page). I suppose another option would be to make the box scrollable and leave it spread out.

Reply:
I agree with you 100% on all of this. I initially added an overflow scroll bar but it seemed to be conflicting with Material UI. I definitely think we should go back and implement both of your suggestions.